src/frontend/efiling-frontend/src/components/page/rush/Rush.js

```diff
@@ -1,42 +1,357 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import PropTypes from "prop-types";
 import Dropzone from "react-dropzone";
 import {
   Radio,
   Callout,
   Button,
-  Input,
   Textarea,
   DatePick,
   Sidecard,
+  Dropdown,
 } from "shared-components";
+import validator from "validator";
 import { getSidecardData } from "../../../modules/helpers/sidecardData";
 import Payment from "../../../domain/payment/Payment";
 
 import "./Rush.scss";
+import { getCountries } from "./RushService";
+import RushDocumentList from "./rush-document-list/RushDocumentList";
+import { Toast } from "../../toast/Toast";
+import { Input } from "../../input/Input";
 
-const calloutText = `Please provide the date of when the direction was made,
-the name of the Judge who made the direction along with any additional details
-you feel are necessary.`;
+const calloutText = `Please provide the date of when the direction was made, the name of the Judge who made the direction along with any additional details you feel are necessary.  `;
 
 const generateInputField = (input, onChange) => (
   <div className="form-child">
     <Input input={input} onChange={onChange} />
   </div>
 );
 
+const formatPhoneNumber = (phoneNumber) => {
```

Comment:
Maybe we can pull this out into a service? I believe there is already a formatting helper module

Reply:
Good point, I didn't have a good look at the helpers. This should be easy to pull into that helper module.